File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/showcase/skins/AbsenceManagementSkin.java
package cz.cvut.fel.matyapav.afandroid.showcase.skins;

import android.content.Context;
import android.view.ViewGroup;

import cz.cvut.fel.matyapav.afandroid.components.skins.DefaultSkin;

/**
 * Created by Pavel on 24.02.2016.
 */
public class AbsenceManagementSkin extends DefaultSkin{

    public AbsenceManagementSkin(Context context) {
        super(context);
    }

    @Override
    public int getInputWidth() {
        return convertDpToPixels(120, getContext());
    }

    @Override
    public int getLabelWidth() {
        return convertDpToPixels(60, getContext());
    }
}

File: examples/ShowcaseWP/ShowcaseWP/skins/AbsenceManagementSkin.cs
using AFWinPhone.builders.skins;

namespace ShowcaseWP.skins
{
    internal class AbsenceManagementSkin : DefaultSkin
    {
        public override int getLabelWidth()
        {
            return 75;
        }

        public override int getInputWidth()
        {
            return 120;
        }

        public override int getLabelFontSize()
        {
            return 18;
        }

        public override int getFieldFontSize()
        {
            return 18;
        }

        public override bool isListItemNameLabelVisible()
        {
            return false;
        }
    }
}